tests/test_download_conda_packages.py

```diff
@@ -0,0 +1,144 @@
+from pathlib import Path
+
+import pytest
+from ruamel.yaml import YAML
+
+from ib_manifest_util import PACKAGE_DIR, TEST_DATA_DIR
+from ib_manifest_util.download_conda_packages import download_packages
+
+small_package_url_and_filename = [
```

Comment:
I would make this a fixture

Reply:
Let me know if the changes are what you were expecting.

Reply:
that'll work. thanks